CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -0,0 +1,61 @@
+## Changes by Kind
```

Comment:
Please add a header, like in older CHANGELOGs

```
Release notes for 1.4.0
[Documentation](https://kubernetes-csi.github.io/)

Changelog since 1.3.0
```

Reply:
oops sorry I just added generated docs. updated. PTAL